oak_functions/gcp/weather_lookup_api_config.yaml

```diff
@@ -0,0 +1,28 @@
+type: google.api.Service
+config_version: 3
+
+#
+# Name of the service configuration.
+#
+name: '*.apigateway.endpoints.oak-ci.cloud.goog'
+
+#
+# API title to appear in the user interface (Google Cloud Console).
+#
+title: Oak Functions gRPC API Gateway
```

Comment:
Is this a generic config that could be used with all Oak Functions applications or is it specific to the `weather_lookup` app? If it is the former, perhaps the file name should be changed?

Reply:
This is specific to the instance being deployed.